### PR TITLE
Module split

### PR DESCRIPTION
**Pull request by Pessimistress**
For #17 

Questions regarding dependencies:
- Is it necessary for core to import probe.gl and webworkify?
- Is it necessary for glb to depend on draco?

### CODE REVIEW

File: aliases.js [hunk 1]
```diff
@@ -22,7 +22,16 @@ const path = require('path');
 
 const ALIASES = {
   'loaders.gl/test': path.resolve(__dirname, './test'),
-  'loaders.gl': path.resolve(__dirname, './modules/core/src')
+  'test-data': path.resolve(__dirname, './test/data'),
+  '@loaders.gl/core': path.resolve(__dirname, './modules/core/src'),
+  '@loaders.gl/draco': path.resolve(__dirname, './modules/draco/src'),
+  '@loaders.gl/glb': path.resolve(__dirname, './modules/glb/src'),
+  '@loaders.gl/gltf': path.resolve(__dirname, './modules/gltf/src'),
```
Comment (ibgreen) [inline aliases.js]: As proposed in the RFC, I think it makes sense to group GLB with glTF. The glTF piece itself is pretty small, we can't do a whole lot with glTF without actually using a WebGL library (which will happen in luma.gl `glTFInstantiator`), so the overhead is pretty small, and GLB is a part of the glTF standard..

File: babel.config.js [hunk 2]
```diff
@@ -48,7 +45,7 @@ CONFIG.es5 = Object.assign({}, CONFIG.default, {
 
 CONFIG.cover = Object.assign({}, CONFIG.default);
 // constant inlining seems to cause problems for nyc
-CONFIG.cover.plugins = ['version-inline', 'istanbul'];
+CONFIG.cover.plugins = ['istanbul'];
```
Comment (ibgreen) [inline babel.config.js]: No version, at least on core library?

Comment (Pessimistress) [reply babel.config.js]: Why do we need this? `deck` and `luma` expose global objects for inspection in the console. For the loaders the version is already defined in package.json.

Comment (Pessimistress) [reply babel.config.js]: Context is this adds additional overhead to publishing (build needs to be run before publish for dist testing then rerun after `lerna publish` to account for the version bump).

**Comment by Pessimistress** [comment]
I am removing all the unused core utils for now. Will do in follow up PRs:

- [ ] Merge `@loaders.gl/glb` into `@loaders.gl/gltf`
- [ ] Add worker processing RFC

**Pull request merged**